### PR TITLE
 Clarifications from Sherlock feedback + few more comments

### PR DESCRIPTION
**Pull request by oldchili**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,15 +63,15 @@ The protocol chooses to override at least the following from https://docs.sherlo
...
-    - not have rebase logic.
-- Note that the lack of any weird feature from the above list of token features does not mean it is supported, these are just examples.
-- ERC-20 tokens - issues stemming from upgradability of these tokens (e.g USDC) are out of scope.
+- The token contracts used in each repository are known in advance and there is no use of arbitrary tokens in any of the contest modules:
+   - In the nst module, apart from the NST token itself only DAI is used (in the DaiNst converter).
+   - In the ngt module, apart from the NGT token itself, only MKR is used (in the MkrNgt converter).
+   - In the snst module, apart from the SNST token, only NST is used (as the deposit token).
+   - In the vote-delegate module only MKR and the existing Chief's IOU tokens are used.
+   - In endgame-toolkit the farm types to be used through different deployment phases are (using rewards/stake notation): NGT/NST, SDAO/NST, NGT/SDAO, NST/LSMKR and SDAO/LSMKR (the last two types are only for lockstake).
+   - In the lockstake module only MKR, NGT and LSMKR are used (plus the two farms types mentioned above).
```
Comment (sunbreak1211) [inline README.md]: Also NST.

File: README.md [hunk 2]
```diff
@@ -84,14 +84,13 @@ The protocol chooses to override at least the following from https://docs.sherlo
 - Discrepancies in handling of zero/non-actionable input amounts (no-op, revert, etc..) are out of scope.
 - Issues in other Makerdao repositories are out of scope, even in case they are relevant to interactions with the contracts in scope. For example - dss, the Chief contract.
 - As the initiation of the endgame components is not considered time-critical, any permissionless action that can cause the spells to revert is not considered a medium or high severity issue (as another spell can be crafted). It can be assumed that no other mission-critical tasks are included in the same spell.
-- Issues where there is damage to the protocol but attack cost exceeds damage caused/funds stolen significantly are considered low severity.
+- Issues where there is damage to the protocol but attack cost exceeds damage caused/funds stolen significantly (50%+ more) are considered low severity.
```
Comment (telome) [inline README.md]: I think "damage caused" and "funds stolen" should be treated differently. The attacker paying `2.5*x` to steal `x` (net cost = `1.5*x`) should be about the same severity as the attacker paying `1.5*x` to grief (but not steal) `x` (net cost = `1.5*x`).
```suggestion
- Issues where there is damage to the protocol/users but the net attack cost exceeds the damage caused significantly (50%+ more) are considered low severity.
```

File: README.md [hunk 3]
```diff
@@ -147,16 +153,18 @@ We are mimicking the old proxy actions behaviour, where we drip for drawing, as
...
 * As with other collaterals, "tip" and "chip" are assumed to be chosen very carefuly, while taking into account the dust parameter and with having in mind incentive farming risks.
 
 ### dss-flappers
 * Inefficiencies due to configurations or oracle frequencies/pricing are out of scope.
 * Slow/Unavoidable delay of withdrawing/redeeming the protocol funds in case of an emergency is known.
 * Possible sandwich attacks and MEV extraction scenarios are known issues. Maker's risk unit is assumed to be setting parameters while having these in mind.
+* The dss-flappers (SBE) solution assumes losing several percentages of funds **by design** during the Uniswap interactions due to slippage, MEV, sudden market movements, sandwich attacks, oracles imprecision and update resolution, etc. Obvioulsy in this specific case the numbers mentioned in the severity definitions (e.g 0.5%) are irrelevant. The history of the SBE can be taken as reference for more than acceptable trading losses. Note that overall the SBE is considered a success in comparison to the previous implemented flap auctions.
```
Comment (telome) [inline README.md]: I feel that the last two sentences may slightly dilute the point made in this paragraph.
```suggestion
* The dss-flappers (SBE) solution assumes losing several percentages of funds **by design** during the Uniswap interactions due to slippage, MEV, sudden market movements, sandwich attacks, oracles imprecision and update resolution, etc. Obvioulsy in this specific case the numbers mentioned in the severity definitions (e.g 0.5%) are irrelevant. ```

File: README.md (hunk 1 above)
Comment (oldchili) [reply README.md]: added.

**Review by telome — APPROVED** [review]
LGTM

File: README.md [hunk 4]
```diff
@@ -84,41 +84,46 @@ The protocol chooses to override at least the following from https://docs.sherlo
...
-- Minor rounding errors leading to missing some fees, getting more fee share compared to someone else, or fees locked in the protocol are considered low severity at best.
-- DoS issues where no funds would be locked or protocol functionality is blocked but one one can still withdraw should be considered low severity at best.
-- Oracles are trusted to provide non-stale and correct information.
+- Minor rounding errors leading to missing some fees, getting more fee share compared to someone else, or fees locked in the protocol are considered low severity at best. For definition of "minor" we use 0.5%.
```
Comment (sunbreak1211) [inline README.md]: Didn't we have to add the minimal values here?

Comment (oldchili) [reply README.md]: done

**Review by telome — APPROVED** [review]

**Review by sunbreak1211 — APPROVED** [review]
LGTM

**Pull request merged**